src/instructlab/model/download.py

```diff
@@ -80,8 +80,10 @@ def download(self):
             if self.ctx.obj is not None:
                 hf_logging.set_verbosity(self.ctx.obj.config.general.log_level.upper())
             files = list_repo_files(repo_id=self.repository, token=self.hf_token)
-            if any(".safetensors" in fname for fname in files):
-                self.download_safetensors()
+            if any(".safetensors" in fname for fname in files) or any(
+                ".bin" in fname for fname in files
```

Comment:
just making sure: In this case, it will download the entire repo even if there's both safetensors _and_ bin. Is this expected/desired behavior?

Reply:
yes. these often overlap